src/lib/moduleMocker.js

```diff
@@ -47,6 +47,48 @@ function getType(ref) {
   return null;
 }
 
+function PlainConstructor() {}
+var plainObject = new PlainConstructor();
+/**
+ * methods on ES6 classes are not enumerable, so they can't
+ * be found with a simple `for (var slot in ...) {`
+ *
+ * scan will return an object with the same shape you would
+ * get with `for ... in` except the non-enumerable methods
+ * are included.
+ */
+ function scan(obj) {
+   var fakeObj = {};
+   var prop;
+   for (prop in obj) {
+     fakeObj[prop] = true;
+   }
```

Comment:
I think we can omit this loop, and go directly to `do-while` considering all properties (and don't do explicit check for `enumerable`)